shared/test/unit/mocks/mock_l20n.js

```diff
@@ -9,7 +9,7 @@
 
     formatValue: function(id, args) {
       return Promise.resolve(
-        stringify(args ? [id, args] : id));
```

Comment:
@stasm - I'm not sure why but it seems that mock_l20n formatValue returned different string than mock_l10n's. I unified the behavior. Let me know if that's ok.